### PR TITLE
merges latest spec with runc

### LINKED ISSUE #528: config: Single, unified config file
Per the changes to the spec... we need to move to a single config file see: 
https://github.com/opencontainers/specs/pull/284

### PR DESCRIPTION
**Pull request by mikebrow**
Closes #528

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: libcontainer/configs/device.go [hunk 1]
```diff
@@ -38,7 +38,13 @@ type Device struct {
 }
 
 func (d *Device) CgroupString() string {
-	return fmt.Sprintf("%c %s:%s %s", d.Type, deviceNumberString(d.Major), deviceNumberString(d.Minor), d.Permissions)
+	var p string
```
Comment (crosbymichael) [inline libcontainer/configs/device.go]: I not sure we need to change this method's logic.  Isn't no perms and invalid config?

Comment (mikebrow) [inline libcontainer/configs/device.go]: The spec moved permissions out... Now these need to come from some new whitelist...

Comment (crosbymichael) [inline libcontainer/configs/device.go]: you have to merge the permission when creating the deivces.  Think of the spec as the View and libcontainer config is the model.  You have to populate it correctly no matter how it is represented in the spec.

Comment (mikebrow) [inline libcontainer/configs/device.go]: Agreed...

**Comment by mikebrow** [comment]
Had to rebase for #525

**Pull request closed without merging**